lib/pages/new_group/new_group_chat_info.dart

```diff
@@ -106,6 +106,8 @@ class NewGroupChatInfoController extends State<NewGroupChatInfo>
         urlAvatar: urlAvatar,
         powerLevelContentOverride: {
           'events': powerLevelManager.getDefaultPowerLevelEventForMember(),
+          'invite': powerLevelManager.getAdminPowerLevel(),
+          'kick': powerLevelManager.getAdminPowerLevel(),
```

Comment:
is it right spec? can you share docs. Thank you

Reply:
https://spec.matrix.org/v1.6/client-server-api/#creation
![image](https://github.com/user-attachments/assets/db8fdebd-fe82-4ca7-b6a7-3eca1ac73cbe)
https://spec.matrix.org/v1.6/client-server-api/#mroompower_levels
![image](https://github.com/user-attachments/assets/b3b8c934-3c35-4e8d-935a-f8f78db4b7e6)